assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -8658,10 +8670,28 @@ private void post_save_assignment(RunData data, String postOrSave) {
                     }
                 }
             }
-            //SAK-45967
-            if (newAssignment || !a.getDraft()) {
-                // post new assignment event since it is fully initialized by now
-                eventTrackingService.post(eventTrackingService.newEvent(AssignmentConstants.EVENT_ADD_ASSIGNMENT, assignmentReference, true));
+            // SAK-45967 && SAK-44623 && SAK-45992
```

Comment:
I'd just get rid of this line. We can work this stuff out from the git history.

Reply:
done